README.md

```diff
@@ -38,6 +41,12 @@ app.use(docs({
     * Package will search for Swagger docs in folder and nested folders.
 * NODE_ENV
     * Package exposes two routes only in specific NODE_ENV's.
+* host
+    * The host of your api. Will fill up the `host` property
```

Comment:
Ik zou eerder zoals de docs hierboven uitleggen wat de property moet bevatten ipv te vermelden dat die comment in de host property gevuld zal worden.
zie de docs van property name of version of baseurl
die omschrijven wat het is, en op 2e regel watof er default waarde is etc.
dat het uiteindelijk ingevuld wordt in die property is een evidentie

en ook de stijl aanhouden,
ik wil me niet uitspreken over API of api bijvoorbeeld, maar in elk geval geen 2 soorten door elkaar. #professionalDocs :)